docs/getting-started/detections-req.asciidoc

```diff
@@ -37,6 +37,70 @@ and restarting {kib}, you must restart all detection rules.
 
 [discrete]
 [[enable-detections-ui]]
+== Enable and access detections
+To use the Detections page, the Detections feature must be enabled and your role must have access to rules and alerts. If your role does not have the cluster and index privileges needed to enable this feature, you can request someone who has these privileges to turn it on for you. The following table describes the required privileges to access the Detections page - including rules and alerts.
```

Comment:
Similar to my previous comment below, I think it's worth explicitly calling out that it's not just these privileges that are necessary to enable the Detections feature, but also that the user with the specified privileges _must visit the Security Application at least once_ for each space they'd like to enable the feature. We've had some [discuss posts](https://discuss.elastic.co/t/cant-access-detections-from-a-different-space/272118/9) where this aspect of the configuration wasn't entirely clear, so would be worth highlighting in a tip/info callout.